README.md

```diff
@@ -1,129 +1,107 @@
-[order books](https://www.ripplecharts.com) allowing anyone to [trade](https://www.rippletrade.com) anything. When two accounts aren’t linked by relationships of trust, the Ripple pathfinding engine considers intermediate links and order books to produce a set of possible paths the transaction can take. When the payment is processed, the liquidity along these paths is iteratively consumed in best-first order.
-![Pathfinding from Dollars to Euro](/images/pathfinding.png)
-
-3. [Consensus](https://www.youtube.com/watch?v=pj1QVb1vlC0) confirms
-transactions in an atomic fashion, without mining, ensuring efficient use of
-resources.
-
-[transact]: https://ripple.com/files/ripple-FIs.pdf
-[build]:    https://ripple.com/build/
-
-[transact.png]:   /images/transact.png
-[build.png]:      /images/build.png
-[contribute.png]: /images/contribute.png
-
-###Join The Ripple Community
-|![Transact][transact.png]|![Build][build.png]|![Contribute][contribute.png]|
-|:-----------------------:|:-----------------:|:---------------------------:|
-|[Transact on the fastest payment infrastructure][transact]|[Build Imaginative Apps][build]|Contribute to the Ripple Protocol Implementation|
-
-#rippled - Ripple P2P server
-
-##[![Build Status](https://travis-ci.org/ripple/rippled.png?branch=develop)](https://travis-ci.org/ripple/rippled)
-
-This is the repository for Ripple's `rippled`, reference P2P server.
-
-###Build instructions:
-* https://ripple.com/wiki/Rippled_build_instructions
-
-###Setup instructions:
-* https://ripple.com/wiki/Rippled_setup_instructions
-
-###Issues
-* https://ripplelabs.atlassian.net/browse/RIPD
-
-### Repository Contents
-
-#### ./bin
+# What is Ripple?
+
+The Ripple Consensus Ledger (RCL) is a secure distributed ledger that uses
+the [consensus process](https://www.youtube.com/watch?v=pj1QVb1vlC0) to
+settle transactions and is maintained by a network of computers over the
+internet, which collectively create Ripple network. Because of its distributed
+nature, it does not require a central operator and offers transaction
+immutability and information redundancy. RCL holds the order book with bid/ask
+offers from payment initiators and market makers. Its path-finding algorithm
+enables it to find the lowest foreign exchange rate across all order books
+and currency pairs.
+
+### Key Features
+#### Distributed
+- Direct account-to-account settlement with no central operator
+- Decentralized global market for competitive FX
+- Transaction immutability and information redundancy
+
+#### Secure
+- Transactions cryptographically signed using ECDSA and ED25519 algorithms
+- Multi-signing capabilities
+
+#### Scalable
+- Capacity to process the world’s cross-border payments volume
+- Easy access to liquidity through a competitive FX marketplace
+
+### XRP Role on Ripple as a Bridge Currency
+Ripple enables banks to settle cross-border payments in real-time, with
+end-to-end transparency, and at lower costs. Banks can choose to either
```

Comment:
> choose to either

Suggest you scratch this whole phrase, so the sentence reads, "Banks can provide liquidity for FX themselves or source it from third-parties.